src/webgui/src/webgui/image_converter.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from std_msgs.msg import String
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+
+import roslib
+import sys
+import rospy
+import cv2
+
+roslib.load_manifest('webgui')
+
+class image_converter:
+
+	def __init__(self):
+		self.bridge = CvBridge()
+		self.image_sub = rospy.Subscriber("image_topic",Image,self.callback, queue_size=10)
+		
+	def callback(self,data):
+		print('image received')
```

Comment:
Let's avoid log statements unless they have useful debug information. If you do need to log something, use rosconsole: http://wiki.ros.org/rospy_tutorials/Tutorials/Logging